packages/backend/src/payment-method/ilp/service.ts

```diff
@@ -43,23 +43,50 @@ async function getQuote(
   deps: ServiceDependencies,
   options: StartQuoteOptions
 ): Promise<PaymentQuote> {
-  const rates = await deps.ratesService
-    .rates(options.walletAddress.asset.code)
-    .catch((_err: Error) => {
-      throw new PaymentMethodHandlerError('Received error during ILP quoting', {
-        description: 'Could not get rates from service',
-        retryable: false
-      })
+  const stopTimerRates = deps.telemetry.startTimer(
+    'ilp_get_quote_rate_time_ms',
+    {
+      callName: 'RateService:rates',
+      description: 'Time to get rates'
+    }
```

Comment:
Should this be directly in the rates service instead? Or instead, called something like `PaymentMethod:ILP:getQuote:getRates`?

Reply:
Hmm fair point. 

We have external timers that wrap function calls from the outside like this, and internal timers that start at the beginning of a function and stop at the end. I think in this case we want the external timer because the goal is to measure `getQuote` and it's parts (thats how this metric is graphed). In other words, we want to measure _this_ rates service call, not all of them.

I take your point on the name. I was thinking about it in the context of the graph, where the `PaymentMethod:ILP:getQuote` would be redundant. However, as soon as we want to time this same method somewhere else the name breaks down, so yeah... looks like that convention (scoped by where its called) is good.

Reply:
> In other words, we want to measure _this_ rates service call, not all of them.

Yes, makes sense. We do have the histogram name `ilp_get_quote_rate_time_ms`, so that is easy to narrow down and trace in the code in any case. 

For the callName, just as long as we can get group metrics from a certain part of the codebase like `PaymentMethod:ILP:*` that's good enough.